conda-store-server/conda_store_server/build.py

```diff
@@ -382,3 +382,43 @@ def build_conda_docker(db: Session, conda_store, build: orm.Build):
         conda_store.log.exception(e)
         append_to_logs(db, conda_store, build, traceback.format_exc())
         raise e
+
+
+def build_constructor_installer(db: Session, conda_store, build: orm.Build):
+    conda_prefix = build.build_path(conda_store)
+
+    settings = conda_store.get_settings(
+        db=db,
+        namespace=build.environment.namespace.name,
+        environment_name=build.environment.name,
+    )
+
+    with utils.timer(
+        conda_store.log, f"creating installer for conda environment={conda_prefix}"
+    ):
+        with tempfile.TemporaryDirectory() as tmpdir:
+            context = action.action_generate_constructor_installer(
+                conda_command=settings.conda_command,
+                specification=schema.CondaSpecification.parse_obj(
+                    build.specification.spec
+                ),
+                installer_dir=pathlib.Path(tmpdir),
+                version=build.build_key,
+            )
```

Comment:
Note for reviewers: the settings/specification part is similar to what's done in `build_conda_environment`.

The rest of this function is similar to `build_conda_pack` or `build_conda_docker ` in this file.